www/src/components/graphiql-embed.js

```diff
@@ -0,0 +1,38 @@
+import React, { useState, useRef, useCallback, useLayoutEffect } from "react"
+
+const VIDEO_RATIO = 2 / 3
+
+const GraphiQLEmbed = ({ title, url, query }) => {
+  const encodedQuery = encodeURIComponent(query)
+
+  const [iframeWidth, setIframeWidth] = useState(0)
+  const iframeRef = useRef()
+
+  const handleResize = useCallback(
+    () => setIframeWidth(iframeRef.current.clientWidth),
+    [iframeRef.current]
+  )
+
+  useLayoutEffect(() => {
+    handleResize()
+    window.addEventListener(`resize`, handleResize)
+
+    return () => {
+      window.removeEventListener(`resize`, handleResize)
+    }
+  }, [iframeRef.current])
+
+  return (
+    <iframe
+      ref={iframeRef}
+      className="graphiql-embed"
+      title={title}
+      src={`${url}?query=${encodedQuery}&explorerIsOpen=false`}
+      width={iframeWidth}
+      height={iframeWidth * VIDEO_RATIO}
+      loading="lazy"
```

Comment:
`loading="lazy"` is only used in some of the queries but not in others. Are we sure we want this behavior across all the embed components?

Reply:
I figured the places where `loading="lazy"` was not being used might be older. Therefore thought it might be a good idea to do this.